master/custom/workers.py

```diff
@@ -144,6 +144,12 @@ def get_workers(settings):
             tags=['linux', 'unix', 'rhel', 'arm', 'arm64', 'aarch64'],
             parallel_tests=40,
         ),
+        cpw(
+            name="diegorusso-aarch64-bigmem",
+            tags=['linux', 'unix', 'ubuntu', 'arm', 'arm64', 'aarch64', 'bigmem'],
+            not_branches=['3.9', '3.10', '3.11', '3.12', '3.13'],
```

Comment:
I'm not sure but can't we use `branches=[MAIN_BRANCH_NAME]`?

Reply:
That is an option, but has different implications.  I suspect this is following how the other bigmem worker is configured, which is set to allow the beta branch to be built without having to change anything extra when it is created.